### ISSUE TITLE
Enhancement: Offline Movies

### ISSUE THREAD

**Issue by Omertron** [YAMJ/yamj-v2#183]
Original [issue 183](https://code.google.com/p/moviejukebox/issues/detail?id=183) created by Omertron on 2008-09-22T02:04:10.000Z:

Lets say you only have 1TB of space for movies but you have more movies 
than that.  So you typically leave movies you have not watched on the 1TB 
and archive (burn to disc) the ones you have seen and remove them from the 
1TB.

What I would like is an offline movie feature - where my movies that are 
offline burned on a disc somewhere still display in the MOvieJukebox.  
Perhaps there is a special &quot;Offline Movies&quot; menu item you can select.  
Perhaps these movies have a special icon on the cover art (indicating they 
are offline) or even all offline cover art is black and white or has a 
special frame around the poster.

Bottom line is I'd like to be able to browse all my movies - online and 
offline (good for when guests come over if they'd like to see a movie I 
dont keep online).
